### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by viren-nadkarni**
Adds issue templates for bug reports and feature requests

### CODE REVIEW

**Comment by dohnto** [comment]
Hmm, not sure whether all the screetshooting/scrolling stuff is related.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (dohnto) [inline .github/ISSUE_TEMPLATE/feature_request.md]: feature?

Comment (weakcamel) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Perhaps "New feature request" ?

**Comment by weakcamel** [comment]
I believe this just needs to be rebased.

> Hmm, not sure whether all the screetshooting/scrolling stuff is related.

True, in our case a copy-paste from the terminal seems more applicable.

**Pull request merged**